Assets/MixedRealityToolkit/Inspectors/Utilities/HandednessInspectorGUI.cs

```diff
@@ -0,0 +1,35 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    public static class HandednessInspectorGUI
+    {
+        private static readonly GUIContent[] HandednessSelections =
+        {
+        new GUIContent("Left Hand"),
+        new GUIContent("Right Hand"),
+        new GUIContent("Both Hands"),
```

Comment:
Please consider using "Any Hand" as "Both Hands" could be read as the controller in intended to be held in two hands (ex: common gamepad controllers)

Reply:
Then the comment on the Both enum entry should be rephrased because it reads:
> The controller is identified as being either left and/or right handed.